src/DataStructures/VariablesHelpers.hpp

```diff
@@ -182,6 +182,38 @@ void add_slice_to_data(const gsl::not_null<Variables<TagsList>*> volume_vars,
   }
 }
 
+namespace OrientVariables_detail {
+
+template <typename TagsList>
+void orient_each_component(
+    const gsl::not_null<Variables<TagsList>*> oriented_variables,
+    const Variables<TagsList>& variables,
+    const std::vector<size_t>& oriented_offset) noexcept {
+  tmpl::for_each<TagsList>(
+      [&oriented_variables, &variables, &oriented_offset](auto tag) {
+        using Tag = tmpl::type_from<decltype(tag)>;
+        auto& oriented_tensor = get<Tag>(*oriented_variables);
+        const auto& tensor = get<Tag>(variables);
```

Comment:
You missed the opportunity to call this one the `disoriented_tensor` :) (This is not a change request)